packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/ClientProvider.cs

```diff
@@ -22,18 +22,25 @@ public class ClientProvider : TypeProvider
         private const string AuthorizationHeaderConstName = "AuthorizationHeader";
         private const string AuthorizationApiKeyPrefixConstName = "AuthorizationApiKeyPrefix";
         private const string ApiKeyCredentialFieldName = "_keyCredential";
+        private const string TokenCredentialScopesFieldName = "AuthorizationScopes";
```

Comment:
is this supposed to be the name of a generated constant? If so, could we rename this to be suffixed with the `ConstName` to follow the existing pattern ?

Reply:
I did not notice there is a pattern which is nice - but this generated field is an array ("scopes") therefore this is not generated as a const - it is something like this:
```csharp
private static readonly string[] AuthorizationScopes = new string[] { "https://api.example.com/.default" };
```
in this case, should this have the `const` suffix?

Reply:
ah I see. Feel free to ignore then.